pkg/utils/string_test.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Generate random string", func() {
+
+	It("does not return the same value on each call", func() {
+		value1, _ := GenerateRandomString(10, 20)
+		value2, _ := GenerateRandomString(10, 20)
+		value3, _ := GenerateRandomString(10, 20)
+
+		Expect(value1).NotTo(Equal(value2))
+		Expect(value1).NotTo(Equal(value3))
+		Expect(value2).NotTo(Equal(value3))
+	})
+
+	It("panics when min is greater than max", func() {
+		Expect(func() { GenerateRandomString(20, 10) }).Should(Panic())
```

Comment:
I almost returned an error, but then decided not to.  The developer should see this panic during the development phase.  If we use this function with varying lengths during runtime then we want to update it  However, I find it hard to see where we would do that.  I can change it if you want.

Reply:
What mistake would someone have to make to see this panic show up in development?

Reply:
So in the case of the Gitlab example, I would pass the wrong numbers in the parameter.  When I go to test the Gitlab flow via unit tests or manually through the CLI it will panic.  It just can't happen.